lib/collection/src/shards/replica_set/mod.rs

```diff
@@ -920,6 +922,8 @@ pub enum ReplicaState {
     // Shard is undergoing recovery by an external node
     // Normally rejects updates, accepts updates if force is true
     Recovery,
+    // TODO
+    Resharding,
```

Comment:
Need a bit of help here. Not sure how to best formulate explanation for this one. :/

Reply:
Thanks, @timvisee!